script/job_generator/versions.py

```diff
@@ -29,7 +29,7 @@
         "12.5",
     ],
     HIPCC: ["5.1", "5.2", "5.3", "5.4", "5.5", "5.6", "5.7", "6.0"],
-    ICPX: ["2024.0"],
+    ICPX: ["2024.2"],
```

Comment:
@SimeonEhrig should we test only the latest version, or all of `2024.0`, `2024.1`, `2024.2` ?

I think testing and supporting only the latest one should be ok, but let me know what you think.

Reply:
Until there is no good reason, we should only test the latest release. It saves a lot of work. We do the same for CUDA on Windows, because there is no user uses an older version.

Maybe in the future, if we use OneAPI cluster, it is required to support older versions.